RevenueCatUI/Templates/LinTemplates/LinTemplateStep1View.swift

```diff
@@ -26,9 +26,9 @@ struct LinTemplateStep1Configuration: Decodable {
 
 extension LinTemplateStep1Configuration {
     static let `default` = LinTemplateStep1Configuration(
-        titleKey: nil,
-        imageNameMacOS: "paywall-first-step-general",
-        imageNameIOS: "paywall-first-step-general",
+        titleKey: "Step1.AuxiliaryDetailsView.Title",
+        imageNameMacOS: "paywall-first-step-general-ios",
+        imageNameIOS: "paywall-first-step-general-macos",
```

Comment:
Are you sure this is correct? Values seem to be inverted.

Reply:
Thanks for catching it! I fixed it.